Source/WebCore/html/shadow/attachmentElementShadow.css

```diff
@@ -24,14 +24,18 @@
 
 div#attachment-container {
 #if (defined(WTF_PLATFORM_IOS_FAMILY) && WTF_PLATFORM_IOS_FAMILY)
-    --icon-size: 72px;
+#if (defined(WTF_PLATFORM_VISION) && WTF_PLATFORM_VISION)
+    --icon-margin: 8px;
+    --information-margin-end: 8px;
```

Comment:
Does this mean we want the save button to be very close to the right edge?

Reply:
yes; the specification is 8px from the edge